src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -81,24 +81,34 @@ public function getDecoratedFactory()
     /**
      * {@inheritdoc}
      */
-    public function createListFromChoices(iterable $choices, $value = null)
+    public function createListFromChoices(iterable $choices, $value = null, $filter = null)
```

Comment:
BC break? missing `callable` type?

Reply:
BC break... But the `callbable` type cannot be added in 6.0 either, because the decorators are decorating it as `$value`: if there are `AbstractOption` or null we cache the choice list, if there are string or property path we convert them to callable using property access.